openhantek/src/docks/VoltageDock.cpp

```diff
@@ -129,3 +155,35 @@ void VoltageDock::setUsed(ChannelID channel, bool used) {
     QSignalBlocker blocker(channelBlocks[channel].usedCheckBox);
     channelBlocks[channel].usedCheckBox->setChecked(used);
 }
+
+int VoltageDock::setProbeGain(ChannelID channel, double probeGain) {
+    if (channel < 0 ||channel >= spec->channels) return -1;
+
+    QSignalBlocker blocker(channelBlocks[channel].probeGainCombobox);
+    int index = (int) (std::find(scope->voltage[channel].probeGainSteps.begin(),
```

Comment:
This was done before with gain as well, that the gain value is stored and the set-Method needed to use `find` to determine the proper index. I prefer the way that the index is stored instead and if the probe value is required, it is looked up like `available_probes[probe_index]`.